src/components/pages/AppDetail/AppDetailContentDetails.tsx

```diff
@@ -95,7 +95,7 @@ export default function AppDetailContentDetails({
               <Button
                 color="secondary"
                 size="small"
-                onClick={() => navigate(-1)}
+                onClick={() => navigate('/appmarketplace')}
```

Comment:
please use `/${PAGES.APP_MARKETPLACE}` for this
As a general rule those kind of strings should only exist once in the application to avoid typos and get better autocompletion support from IDE - it also reduces the bundle size if we don't repeat typing everything

Reply:
@oyo Made the changes